docs/src/user/manual/home.md

```diff
@@ -26,3 +26,30 @@ The crisp chat interface allows you to directly contact administrators of the pl
 |   |   |
  --- | --- |
 | ![interface](./img/crisp_chat_icon.png) | ![Crisp_chat_interface](./img/crisp_chat.png) |
+
+# Home Page User Manual (After Login)
```

Comment:
@ketankartoza There should only ever be one level 1 heading in a document file otherwise there are build issues.
```suggestion
## Home Page User Manual (After Login)
```

Reply:
![image](https://github.com/kartoza/SAEOSS-Portal/assets/100694811/da183578-800d-4047-b6a2-35e9e75a4dcc)